app/controllers/admin/court_applications_controller.rb

```diff
@@ -33,15 +33,22 @@ def create
 
     def show
       @court_application = CourtApplication.find(params[:id])
+      @court_application_hearings = @court_application.court_hearings
+
+      if @court_application.prosecution_case.present?
+        @prosecution_case = ProsecutionCase.find(@court_application.prosecution_case)
+        @prosecution_case_hearings = @prosecution_case.hearings
+      end
     end
 
     def edit
       @court_application = CourtApplication.find(params[:id])
       # checks for existing court applications for prosecution_case as this is a new link table used only for appeals work.
       if @court_application.prosecution_case.present?
-        prosecution_case = ProsecutionCase.find(@court_application.prosecution_case)
-        @defendants = prosecution_case.defendants
-        @hearings = prosecution_case.hearings
+        @court_application_hearings = @court_application.court_hearings
```

Comment:
None of these ivars get used as far as I can tell - they're not needed by the `edit` template. If they _were_ needed, we'd also need to add them when calling `render :edit` in `update` below. But as they're not, what's the point of adding them here?